contracts/StaderConfig.sol

```diff
@@ -67,6 +67,8 @@ contract StaderConfig is IStaderConfig, AccessControlUpgradeable {
     //Roles
     bytes32 public constant override MANAGER = keccak256("MANAGER");
     bytes32 public constant override OPERATOR = keccak256("OPERATOR");
+    bytes32 public constant override ROLE_SD_REWARD_ENTRY = keccak256("ROLE_SD_REWARD_ENTRY");
```

Comment:
i believe we should also expose a write function to grant these Roles by MANAGER, as ADMIN is an external multisig and needn't be pulled incase we want to grant/revoke this role .
----
Else we can have ROLE_SD_REWARD_ENTRY_ACCOUNT an 1/n multisig, and approver also an multisig, so as there is little to no probability of changing thses addresses

Reply:
To maintain traceability, I would keep it as it is. I agree if we have multisig for the new set of permissions. WDYT @galacticminter